src/app/hooks/queries/useBnsName.ts

```diff
@@ -43,7 +43,7 @@ export const useBNSResolver = (
         setAssociatedAddress('');
       }
     })();
-  }, [recipientAddress]);
+  }, [recipientAddress, network, currencyType, selectedNetwork, walletAddress]);
```

Comment:
what is the difference between network and selectedNetwork?

Reply:
One is NetworkSettings, the other is StacksNetwork